js/wasm_exec.js

```diff
@@ -0,0 +1,464 @@
+// NOTE: This package is using esbuild with a plugin that inlines WASM.
+//    Because of this the "fs" module is no longer needed, and the polyfill
+//    for that has been removed, along with any need to detect the existence
+//    of a global `require`.
+
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// This file has been modified for use by the TinyGo compiler.
+//
+// This file has been modified for use by Tableland.
+
+(() => {
+    // Map multiple JavaScript environments to a single common API,
```

Comment:
@brunocalza Looks like from this line to the bottom of the file has an extra 2 spaces of indentation when compared to the old repo.